### PR TITLE
New Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-22846__20201028.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+22846,CC_scale_factor1,0.45,constant; CP02PMUO-00016
+22846,CC_scale_factor2,0.45,constant
+22846,CC_scale_factor3,0.45,constant
+22846,CC_scale_factor4,0.45,constant
+22846,CC_bin_size,800.0,in cm
+22846,CC_dist_first_bin,1695.0,in cm
+22846,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSL/CGINS-ADCPSL-22846__20201028.csv:8]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-20496__20201028.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20496,CC_scale_factor1,0.45,constant; CP03ISPM-00004
+20496,CC_scale_factor2,0.45,constant
+20496,CC_scale_factor3,0.45,constant
+20496,CC_scale_factor4,0.45,constant
+20496,CC_bin_size,400.0,in cm
+20496,CC_dist_first_bin,901.0,in cm
+20496,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPTG/CGINS-ADCPTG-20496__20201028.csv:8]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-24544__20201028.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24544,CC_scale_factor1,0.45,constant; CP01CNPM-00004
+24544,CC_scale_factor2,0.45,constant
+24544,CC_scale_factor3,0.45,constant
+24544,CC_scale_factor4,0.45,constant
+24544,CC_bin_size,400.0,in cm
+24544,CC_dist_first_bin,901.0,in cm
+24544,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPTG/CGINS-ADCPTG-24544__20201028.csv:8]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00003__20201028.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR003,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00013
+LGR003,CC_height_of_air_humidity_measurement_m,4.255,
+LGR003,CC_height_of_air_temperature_measurement_m,4.255,
+LGR003,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR003,CC_jcool,1,Constant
+LGR003,CC_jwarm,1,Constant
+LGR003,CC_zinvpbl,600,Constant
+LGR003,CC_use_velpt,1,Constant
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00003__20201028.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00007__20201028.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR007,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00013
+LGR007,CC_height_of_air_humidity_measurement_m,4.255,
+LGR007,CC_height_of_air_temperature_measurement_m,4.255,
+LGR007,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR007,CC_jcool,1,Constant
+LGR007,CC_jwarm,1,Constant
+LGR007,CC_zinvpbl,600,Constant
+LGR007,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00007__20201028.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00033__20201028.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR033,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00012
+LGR033,CC_height_of_air_humidity_measurement_m,4.255,
+LGR033,CC_height_of_air_temperature_measurement_m,4.255,
+LGR033,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00033__20201028.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00038__20201028.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR038,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP04OSSM-00012
+LGR038,CC_height_of_air_humidity_measurement_m,4.255,
+LGR038,CC_height_of_air_temperature_measurement_m,4.255,
+LGR038,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR038,CC_jcool,1,Constant
+LGR038,CC_jwarm,1,Constant
+LGR038,CC_zinvpbl,600,Constant
+LGR038,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00038__20201028.csv:9]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20200911.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
...3.29, 274.1, 274.92, 275.73, 276.54, 277.36, 278.17, 278.98, 279.8, 280.61, 281.42, 282.24, 283.05, 283.87, 284.68, 285.49, 286.31, 287.12, 287.93, 288.75, 289.56, 290.37, 291.19, 292, 292.81, 293.63, 294.44, 295.25, 296.07, 296.88, 297.69, 298.51, 299.32, 300.13, 300.95, 301.76, 302.57, 303.38, 304.2, 305.01, 305.82, 306.64, 307.45, 308.26, 309.07, 309.89, 310.7, 311.51, 312.32, 313.13, 313.95, 314.76, 315.57, 316.38, 317.19, 318, 318.82, 319.63, 320.44, 321.25, 322.06, 322.87, 323.68, 324.49, 325.3, 326.11, 326.92, 327.73, 328.55, 329.36, 330.17, 330.98, 331.78, 332.59, 333.4, 334.21, 335.02, 335.83, 336.64, 337.45, 338.26, 339.07, 339.88, 340.68, 341.49, 342.3, 343.11, 343.92, 344.72, 345.53, 346.34, 347.14, 347.95, 348.76, 349.57, 350.37, 351.18, 351.98, 352.79, 353.6, 354.4, 355.21, 356.01, 356.82, 357.62, 358.43, 359.23, 360.04, 360.84, 361.64, 362.45, 363.25, 364.05, 364.86, 365.66, 366.46, 367.27, 368.07, 368.87, 369.67, 370.47, 371.28, 372.08, 372.88, 373.68, 374.48, 375.28, 376.08, 376.88, 377.68, 378.48, 379.28, 380.08, 380.88, 381.68, 382.47, 383.27, 384.07, 384.87, 385.67, 386.46, 387.26, 388.06, 388.85, 389.65, 390.45, 391.24, 392.04, 392.83, 393.63, 394.42, 395.22]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01093__20200911.csv:8]: GI01SUMO-6 – Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20200911.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
...
....36, 273.18, 273.99, 274.8, 275.61, 276.42, 277.24, 278.05, 278.86, 279.67, 280.48, 281.3, 282.11, 282.92, 283.73, 284.54, 285.36, 286.17, 286.98, 287.79, 288.6, 289.42, 290.23, 291.04, 291.85, 292.66, 293.48, 294.29, 295.1, 295.91, 296.72, 297.53, 298.34, 299.16, 299.97, 300.78, 301.59, 302.4, 303.21, 304.02, 304.83, 305.65, 306.46, 307.27, 308.08, 308.89, 309.7, 310.51, 311.32, 312.13, 312.94, 313.75, 314.56, 315.37, 316.18, 316.99, 317.8, 318.61, 319.42, 320.23, 321.04, 321.85, 322.66, 323.47, 324.27, 325.08, 325.89, 326.7, 327.51, 328.32, 329.13, 329.93, 330.74, 331.55, 332.36, 333.16, 333.97, 334.78, 335.59, 336.39, 337.2, 338.01, 338.81, 339.62, 340.43, 341.23, 342.04, 342.84, 343.65, 344.46, 345.26, 346.07, 346.87, 347.68, 348.48, 349.29, 350.09, 350.9, 351.7, 352.5, 353.31, 354.11, 354.91, 355.72, 356.52, 357.32, 358.13, 358.93, 359.73, 360.53, 361.33, 362.14, 362.94, 363.74, 364.54, 365.34, 366.14, 366.94, 367.74, 368.54, 369.34, 370.14, 370.94, 371.74, 372.54, 373.34, 374.14, 374.94, 375.73, 376.53, 377.33, 378.13, 378.92, 379.72, 380.52, 381.31, 382.11, 382.91, 383.7, 384.5, 385.29, 386.09, 386.88, 387.68, 388.47, 389.26, 390.06, 390.85, 391.65, 392.44, 393.23, 394.02]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01092__20200911.csv:8]: GI01SUMO-6 – Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20201014.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
....92, 274.74, 275.55, 276.37, 277.18, 278, 278.81, 279.63, 280.44, 281.26, 282.07, 282.88, 283.7, 284.51, 285.33, 286.14, 286.96, 287.77, 288.59, 289.4, 290.22, 291.03, 291.85, 292.66, 293.48, 294.29, 295.11, 295.92, 296.74, 297.55, 298.37, 299.18, 299.99, 300.81, 301.62, 302.44, 303.25, 304.07, 304.88, 305.69, 306.51, 307.32, 308.14, 308.95, 309.76, 310.58, 311.39, 312.21, 313.02, 313.83, 314.65, 315.46, 316.27, 317.09, 317.9, 318.71, 319.53, 320.34, 321.15, 321.96, 322.78, 323.59, 324.4, 325.21, 326.03, 326.84, 327.65, 328.46, 329.27, 330.09, 330.9, 331.71, 332.52, 333.33, 334.14, 334.95, 335.77, 336.58, 337.39, 338.2, 339.01, 339.82, 340.63, 341.44, 342.25, 343.06, 343.87, 344.68, 345.49, 346.3, 347.11, 347.91, 348.72, 349.53, 350.34, 351.15, 351.96, 352.76, 353.57, 354.38, 355.19, 355.99, 356.8, 357.61, 358.42, 359.22, 360.03, 360.84, 361.64, 362.45, 363.25, 364.06, 364.86, 365.67, 366.47, 367.28, 368.08, 368.89, 369.69, 370.5, 371.3, 372.1, 372.91, 373.71, 374.51, 375.32, 376.12, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.73, 382.53, 383.33, 384.13, 384.93, 385.73, 386.53, 387.33, 388.13, 388.93, 389.73, 390.53, 391.32, 392.12, 392.92, 393.72, 394.51, 395.31, 396.11]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01065__20201014.csv:8]: ISSM-12 – Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20201014.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20201014.csv:8]: OSSM-12 – Looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20201014.csv (hunk 11 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01107__20201014.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01093__20200911.csv (hunk 8 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01093__20200911.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01092__20200911.csv (hunk 9 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01092__20200911.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20201014.csv (hunk 10 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01065__20201014.csv:8]: Looks good

File: calibration/METBKA/CGINS-METLGR-00038__20201028.csv (hunk 7 above)
Comment (batryn) [reply calibration/METBKA/CGINS-METLGR-00038__20201028.csv:9]: Looks good

File: calibration/METBKA/CGINS-METLGR-00033__20201028.csv (hunk 6 above)
Comment (batryn) [reply calibration/METBKA/CGINS-METLGR-00033__20201028.csv:9]: Looks good

File: calibration/METBKA/CGINS-METLGR-00007__20201028.csv (hunk 5 above)
Comment (batryn) [reply calibration/METBKA/CGINS-METLGR-00007__20201028.csv:9]: Looks good

File: calibration/METBKA/CGINS-METLGR-00003__20201028.csv (hunk 4 above)
Comment (batryn) [reply calibration/METBKA/CGINS-METLGR-00003__20201028.csv:9]: Looks good

File: calibration/ADCPTG/CGINS-ADCPTG-24544__20201028.csv (hunk 3 above)
Comment (batryn) [reply calibration/ADCPTG/CGINS-ADCPTG-24544__20201028.csv:8]: Looks good

File: calibration/ADCPTG/CGINS-ADCPTG-20496__20201028.csv (hunk 2 above)
Comment (batryn) [reply calibration/ADCPTG/CGINS-ADCPTG-20496__20201028.csv:8]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-22846__20201028.csv (hunk 1 above)
Comment (batryn) [reply calibration/ADCPSL/CGINS-ADCPSL-22846__20201028.csv:8]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Pull request merged**